profiler/src/ProfilerEngine/Datadog.Profiler.Native/DllMain.cpp

```diff
@@ -61,23 +61,23 @@ bool IsProfilingEnabled(Configuration const& configuration)
 
     Log::Info(".NET Profiler deployment mode: ", to_string(deploymentMode));
 
-
     if (enablementStatus == EnablementStatus::NotSet || enablementStatus == EnablementStatus::SsiEnabled)
     {
         auto isSsiDeployed = deploymentMode == DeploymentMode::SingleStepInstrumentation;
         if (isSsiDeployed)
         {
-            Log::Info(".NET Profiler is enabled using Single Step Instrumentation limited activation.");
+            Log::Info(".NET Profiler is deployed using Single Step Instrumentation.");
```

Comment:
Should we still log that EnvironmentVariables::ProfilerEnabled was not set, and so profiler will be disabled?

Reply:
> Should we still log that EnvironmentVariables::ProfilerEnabled was not set, and so profiler will be disabled?

This is done line 80/81
```
auto isEnabled = (enablementStatus == EnablementStatus::ManuallyEnabled);
Log::Info(".NET Profiler is ", std::boolalpha, (isEnabled ? "enabled." : "disabled."));
```
even though we don't explain why (i.e. missing env var or explicit =false)

The "full" implementation will be done in the next PR that will support SSI enablement

BTW, this file might have been committed too early from that PR  :^(

Reply:
Oh yeah, looks good, thanks!